forge/forge/op_repo/__init__.py

```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Operator repository datatypes
+#
```

Comment:
Similar for previous MR, let's separate licence header and function comments.

Reply:
Resolved in #331